### PR TITLE
Ok/hello world

### PR DESCRIPTION
**Pull request by Olgamalyuta**
1ST test

### CODE REVIEW

File: src/test/java/school/redrover/HelloWorldTest.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import org.testng.annotations.Test;
+
+public class HelloWorldTest {
+
+    @Test
+    public void getUrlKovalenko (){
+        for (int i=0;i<3;i++){
+            WebDriver driver = new ChromeDriver();
+            driver.get("http:/google.com");
+            driver.quit();
```
Comment (verafes) [inline src/test/java/school/redrover/HelloWorldTest.java:14]: предполагаю, что в названии теста должно быть слово test, например: testHomePage()
иначе не пройдет CI

Comment (matsstang) [reply src/test/java/school/redrover/HelloWorldTest.java:21]: but what reason of this code?
what it has to do?

Comment (Olgamalyuta) [reply src/test/java/school/redrover/HelloWorldTest.java:21]: Nothing, just practice with git

Comment (matsstang) [reply src/test/java/school/redrover/HelloWorldTest.java:21]: ok

**Review by matsstang — APPROVED** [review]

**Review by GelikaUS — APPROVED** [review]

**Comment by matsstang** [comment]
your pull request is already approved
you have only to merge your pull request

**Pull request merged**